packages/api/src/graphql/mutation.public.ts

```diff
@@ -85,14 +86,17 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
 
     // Comment
     // =======
-    createComment: {
+    addPublicComment: {
```

Comment:
here we are in the public mutation so i would call the mutation just `addComment`

Reply:
fixed